setup_tests.sh

```diff
@@ -10,7 +15,7 @@ else
 fi
 
 echo "Checking if OSM + GTFS test files exist; downloading if needed"
-if [[ ! -f "./web/test-data/mini_nor_cal.osm.pbf" ]]; then
+if [[ ! -f "./web/test-data/micro_nor_cal.osm.pbf" ]]; then
```

Comment:
Note: I also edited this line while addressing PR comments, because we're deleting the mini nor cal OSM after doing the micro cutout, so the better check for "have we gotten OSM ready yet?" is seeing if the micro OSM cutout exists